docs/samples/Microsoft.ML.Samples/Trainers.cs

```diff
@@ -74,5 +74,122 @@ public static void SdcaRegression()
             Console.WriteLine($"RMS - {metrics.Rms}");  // 4.924493
             Console.WriteLine($"RSquared - {metrics.RSquared}");  // 0.565467
         }
+
+        public static void FastTreeRegression()
+        {
+            // Downloading a regression dataset from github.com/dotnet/machinelearning
+            // this will create a housing.txt file in the filsystem this code will run
+            // you can open the file to see the data. 
+            string dataFile = SamplesUtils.DatasetUtils.DownloadHousingRegressionDataset();
+
+            // Creating the ML.Net IHostEnvironment object, needed for the pipeline
+            var env = new LocalEnvironment(seed: 0);
+
+            // Creating the ML context, based on the task performed.
+            var regressionContext = new RegressionContext(env);
+
+            // Creating a data reader, based on the format of the data
+            var reader = TextLoader.CreateReader(env, c => (
+                        label: c.LoadFloat(0),
+                        features: c.LoadFloat(1, 6)
+                    ),
+                separator: '\t', hasHeader: true);
+
+            // Read the data, and leave 10% out, so we can use them for testing
+            var data = reader.Read(new MultiFileSource(dataFile));
+            var (trainData, testData) = regressionContext.TrainTestSplit(data, testFraction: 0.1);
+
+            // The predictor that gets produced out of training
+            LinearRegressionPredictor pred = null;
+
+            // Create the estimator
+            var learningPipeline = reader.MakeNewEstimator()
+                .Append(r => (r.label, score: regressionContext.Trainers.FastTree(
```

Comment:
>FastTree [](start = 73, length = 8)

you can use more parameters, if you like.

Reply:
Yes, having additional hyperparameters in the examples invites users to try different values. While our default hyperparameters are good (honed on many datasets), gains can be had by exploring alternative values. 

For FastTree, I'd recommend including the defaults--`numTrees: 100, numLeaves: 20, minDocumentsInLeafs: 10, learningRate: 0.2`.

If we want to be even more helpful to the user, we can suggest ranges within the examples:
```suggestion
                .Append(r => (r.label, score: regressionContext.Trainers.FastTree(
                               numTrees: 100, // try: (int) 20-2000
                               numLeaves: 20, // try: (int) 2-128
                               minDocumentsInLeafs: 10, // try: (int) 1-100
                               learningRate: 0.2, // try: (float) 0.025-0.4        
```

As I'm typing from memory, the syntax maybe incorrect.